ui/overviewTab/overviewTab.scss

```diff
@@ -1,169 +1,162 @@
+/* ---------- Overview tab ------------- */
+.overview-tab {
+  > .horizontal-flex {
     display: flex;
-    flex-direction: row;
-    align-items: center;
+    gap: 16px;
   }
 
-  div[data-groupname] .equipmentname {
-    text-align: right;
-  }
-
-  .table-header, div[data-groupname] {
-    span {
-      display: inline-block;
-    }
-
-    >div {
-      border-width: 0px 0px 1px 0px;
-      border-color: $border-underline-color;
-      border-style: solid;
-      margin-bottom: 2px;
-    }
-
-    .equipmentname {
+  .advantage-overview {
+    $advantage-grid-template: getResponsiveGridWidths(128px);
+    @include epic-table-grid('advantage', $advantage-grid-template);
+    @include epic-table-grid('extraadvantage', $advantage-grid-template);
+
+    .advantage-row {
+      display: flex;
+      justify-content: space-between;
+      width: 100%;
+      @include underline-row();
+      /* If the name wraps, make the lines close. */
       line-height: 1;
-      padding-top: 2px;
-      padding-bottom: 2px;
-      margin-right: 4px;
-      flex: 0 0 100px;
-      width: 100px;
-    }
-
-    .equipmentdamage {
-      width: 60px;
-      text-align: center
-    }
-
-    .equipmentrange {
-      width: 50px;
-      text-align: center;
+      div, span {
+        text-align: left !important;
+      }
     }
 
-    .equipmentnotes {
-      margin-left: 90px;
-      margin-bottom: 2px;
-      width: 150px;
+    .extraadvantage-row {
+      @include underline-row();
+      /* If the name wraps, make the lines close. */
       line-height: 1;
-      flex: 1 1;
+      text-align: left !important;
     }
-  }
-}
 
-/* ------------ Skills Overview -------------- */
-.skills-overview {
-  .table-header {
-    display: flex;
-    flex-direction: row;
-    div {
-      text-align: center;
+    >.conditional-something> :first-child[value="-1"]~* {
+      color: red;
     }
   }
 
-  .skill-overview-row-container {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    border-width: 0px 0px 1px 0px;
-    border-color: $border-underline-color;
-    border-style: solid;
-  }
-
-  .isdiscipline[value="1"]~* {
-    display: none;
-  }
 
-  div[data-groupname] .skillname {
-    text-align: right;
-    margin-right: 4px;
-  }
-
-  div[data-groupname] .skillability {
-    margin-left: -4px;
-    margin-right: -4px;
+  /* ------------ Feat Overview -------------- */
+  .feat-overview {
+    $feat-grid-template: getResponsiveGridWidths(24px, 96px);
+    @include epic-table-grid('feat', $feat-grid-template);
+
+    >div[data-groupname] {
+      >div {
+        @include underline-row();
+      }
+      .feat-row {
+        &.featextraordinaryletter {
+          text-align: center;
+        }
+        &.featname {
+          text-align: left;
+        }
+      }
+    }
   }
 
-
-  .table-header, div[data-groupname] {
-    span {
-      display: inline-block;
+  /* ------------ Weapons Overview -------------- */
+  .weapons-overview {
```

Comment:
I haven't converted weapons yet.
It should be easy enough, but have to work through the notes block.